src/components/TheHeader.vue

```diff
@@ -26,53 +27,126 @@
         </svg>
       </a>
     </div>
+    <burger />
   </navbar>
 </template>
 
 <script>
+import burger from './TheBurger.vue';
+
 export default {
+  components: {
+    burger
+  },
   data() {
     return {
-      userName: 'Trushik Semyon'
+      userName: 'Trushik Semyon',
+      switchToggle: false
     };
+  },
+
+  methods: {
+    openLinks(switchToggle) {
+      this.switchToggle = !switchToggle;
+    }
   }
 };
 </script>
-<style lang="scss">
-@import '../style/variables.scss';
+<style scoped lang="scss">
+@import "../style/abstracts/mixins";
+@import "../style/abstracts/variables";
 .header {
-    align-items: center;
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 2rem 3rem;
+  a {
+    color: var(--all-text-white);
+    text-decoration: none;
+    font-size:3rem;
```

Comment:
Maybe need to use prettier also in format script or check if there some rules in eslint ( but I'm not sure if it exists for CSS) 
```suggestion
    font-size: 3rem;
```